app/Jobs/SyncPackagePackagistData.php

```diff
@@ -46,14 +64,22 @@ public function handle()
             return;
         }
 
-        Package::withoutSyncingToSearch(function () use ($packagistData) {
+        Package::withoutSyncingToSearch(function () use ($packagistData, $novaVersion) {
             $this->package->update([
                 'packagist_downloads' => Arr::get($packagistData, 'package.downloads.total', 0) ?: 0,
                 'github_stars' => Arr::get($packagistData, 'package.github_stars', 0) ?: 0,
                 'repo_url' => $packagistData['package']['repository'],
+                'nova_version' => $novaVersion ?? null,
```

Comment:
I don't think we need ` ?? null` here right?

```suggestion
                'nova_version' => $novaVersion,
```

Reply:
Sure, will make the changes tonight 😃